website/static/js/profile.js

```diff
@@ -555,8 +609,44 @@ var SocialViewModel = function(urls, modes) {
 SocialViewModel.prototype = Object.create(BaseViewModel.prototype);
 $.extend(SocialViewModel.prototype, SerializeMixin.prototype, TrackedMixin.prototype);
 
-var ListViewModel = function(ContentModel, urls, modes) {
+SocialViewModel.prototype.serialize = function() {
+    var serializedData = ko.toJS(this);
+    
+    function removeBlankValues(value) {
```

Comment:
This code can also be removed if the code is refactored to not allow blank values into `profileWebsites`.